indico/modules/events/contributions/controllers/management.py

```diff
@@ -60,6 +62,22 @@
 from MaKaC.webinterface.rh.conferenceModif import RHConferenceModifBase
 
 
+def _handle_legacy_description(field, obj):
+    """Check if the object in question is using an HTML description and convert it.
+
+       The description will be automatically converted to Markdown and a warning will
+       be shown next to the field.
+
+       :param field: the WTForms field to be checked
+       :param obj: the object whose render mode/description will be checked
+    """
+    if obj.render_mode == RenderMode.html:
+        field.warning = _("This description has been automatically converted from HTML to Markdown. "
+                          "Please double-check that it's properly displayed.")
+        ht = HTML2Text()
+        field.data = ht.handle(unicode(obj.description))
```

Comment:
`obj._description` might be safer to get the raw html content without any preprocessing by the description wrapper